docs/source/usage.rst

```diff
@@ -8,6 +8,12 @@ Usage
 Tailscale
 =========
 
+.. warning::
+
+   It is not recommended to use Tailscale. Tailscale is a VPNaaS provider
+   based on Wireguard. You don't know what exactly Tailscale does and
+   what's in their client.
```

Comment:
If this is true, it should not be used at all. Maybe then rather drop it from the docs completely and possibly deprecate it?